package.json

```diff
@@ -46,7 +46,7 @@
     "@types/utif": "^3.0.1",
     "@typescript-eslint/eslint-plugin": "^4.19.0",
     "@typescript-eslint/parser": "^4.18.0",
-    "chromedriver": "^99.0.0",
+    "chromedriver": "^100.0.0",
```

Comment:
Do we actually need this dependency?

Reply:
We do need it, and it's nowhere to be found in jest-browserstaci-automate's package.json.

Reply:
It's probably best listed as an optional dep (although we don't list it on Manage at all) as you'd only need it if running the visual tests locally?